spec/controllers/v1.0/requests_controller_spec.rb

```diff
@@ -166,17 +255,64 @@
     let(:item) { { 'disk' => '100GB' } }
     let(:valid_attributes) { { :requester_name => '1234', :name => 'Visit Narnia', :content => item, :description => 'desc' } }
 
-    context 'when request attributes are valid' do
+    context 'admin role when request attributes are valid' do
+      let(:access_obj) { instance_double(RBAC::Access, :accessible? => true, :admin? => true, :approver? => false, :owner? => false) }
+      before do
+        ENV['AUTO_APPROVAL'] = 'y'
```

Comment:
@hsong-rh Instead of setting the environment variable directly can we use this approach
https://github.com/ManageIQ/catalog-api/blob/cee85e285bec837b21cc0fe6723097bdfbd2539c/spec/lib/dev_credentials_spec.rb#L4
Using the gem 'climate_control'